packages/forms-web-app/src/dynamic-forms/has-appeal-form/journey.js

```diff
@@ -89,7 +89,12 @@ class HasAppealFormJourney extends Journey {
 			new Section('Application', 'application')
 				.addQuestion(questions.uploadOriginalApplicationForm)
 				.addQuestion(questions.uploadChangeOfDescriptionEvidence)
-				.addQuestion(questions.uploadApplicationDecisionLetter)
+				.addQuestion(questions.uploadApplicationDecisionLetter),
+			new Section('Your application', 'your-application')
```

Comment:
Nooooooo not an inevitable conflict 
![reeee](https://github.com/Planning-Inspectorate/appeal-planning-decision/assets/35689158/fd13f04e-14d4-4f8d-9d54-438a453346b2)

Reply:
Oops, I've put the section in the right place following a rebase